src/pages/Reader.tsx

```diff
@@ -208,19 +251,47 @@ class Reader extends React.Component<any, State> {
   }
 
   handleMouseEvent = (e: MouseEvent<HTMLDivElement>) => {
-    console.log(this.state.imageFiles)
     if (e.pageX < window.innerWidth * 0.5) {
       this.previousPage();
     } else {
       this.nextPage();
     }
+  }
+
+  setKeyBindings = (newKeyBindings: KeyBindings) => {
+    this.setState(curState => ({
+      ...curState,
+      keyBindings: newKeyBindings
+    }));
+    localStorage.setItem("carrotKeyBindings", JSON.stringify(newKeyBindings))
+  }
+
+  handleHotKeys = (event: KeyboardEvent) => {
+    const key = event.key.toUpperCase();
```

Comment:
why `toUpperCase`

Reply:
Oh no wonder if think it's uppercase because they're capitals. I meant for them to just be the key, no need for shift. Should they be displayed as lowercase instead

Reply:
yeah display them as lowercase